frontend/.eleventy.js

```diff
@@ -2,10 +2,6 @@
  * sri sri guru gaurangau jayatah
  */
 
-require('dotenv').config({
-  path: `${__dirname}/.env.local`,
-});
-
```

Comment:
Why have you removed this code?

Reply:
we are not using `dotenv` on anything as of now, we might need this configuration later.